docs/Sonarr/V3/Sonarr-Release-Profile-RegEx.md

```diff
@@ -82,6 +82,7 @@ The Number between the **[**brackets**]** are the scores the release name will g
 
 ### Release Sources (Streaming Service)
 
+<!-- [trash_id: a0e7774a471e041d4f1111e0690244d0] -->
```

Comment:
I recommend moving this to line 79 (below `##First Release Profile`). Each new section starts some new parsing logic, so I think keeping it closer to the top level section will be more helpful.